migrations/sql/R__stored_procedure_for_mine_information.sql

```diff
@@ -63,7 +64,8 @@ CREATE OR REPLACE FUNCTION transfer_mine_information() RETURNS void AS $$
                                     WHEN mms.mmsmin.mine_typ = ANY('{Q,CM,SG}'::text[]) THEN 'BCL'
                                     ELSE NULL
                                 END,
-                deleted_ind    = LOWER(mms.mmsmin.reg_cd) = 'x' OR LOWER(mms.mmsmin.mine_nm) LIKE '%delete%' OR LOWER(mms.mmsmin.mine_nm) LIKE '%reuse%'
+                deleted_ind    = LOWER(mms.mmsmin.reg_cd) = 'x' OR LOWER(mms.mmsmin.mine_nm) LIKE '%delete%' OR LOWER(mms.mmsmin.mine_nm) LIKE '%reuse%',
+                mms_alias      = mms.mmsmin.alias 
```

Comment:
Will this pull across the white space in the alias column stored in MMS? Or does/should it trim off the blanks? Not a deal breaker...